nav2_planner/src/planner_server.cpp

```diff
@@ -62,6 +63,8 @@ PlannerServer::PlannerServer(const rclcpp::NodeOptions & options)
     }
   }
 
+  get_parameter("is_path_valid_check_footprint", is_path_valid_check_footprint_);
```

Comment:
Is there a standard parameter name we use for checking full footprint? If nothing else, drop the `is_` up front to shorten it a hair

Reply:
^ would be potentially be valuable to use the costmap's information about whether its a radius or a footprint to make that call instead of making it a parameter?